pom.xml

```diff
@@ -56,7 +56,11 @@
             <artifactId>httpclient</artifactId>
             <version>4.5</version>
         </dependency>
-
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>23.6-android</version>

```

Comment:
Please use
<dependency>
    <groupId>com.google.guava</groupId>
    <artifactId>guava</artifactId>
    <version>23.6-jre</version>
</dependency>

Reply:
Hi @JanGurda :)

The point of this PR is to not force `Java 8` where we want to use this library as a dependency.

Extracted from the [guava documentation](https://github.com/google/guava#guava-google-core-libraries-for-java):

>Guava comes in two flavors.
>
>- The JRE flavor requires JDK 1.8 or higher.
>- If you need support for JDK 1.7 or Android, use the Android flavor. You can find the Android Guava source in the android directory.

So, if we added here the `jre` version, this would still require _java 8_. So, given that the goal of this PR is to remove the requirement of using _java 8_ from where we want to add this library as a dependency, that's why I added the _android_ flavor.

Does it make sense?

Reply:
OK then. I didn't know that.